components/social/org.wso2.carbon.social.core/src/main/java/org/wso2/carbon/social/core/service/SocialActivityService.java

```diff
@@ -193,12 +221,29 @@ public boolean removeActivity(int activityId, String userId) throws SocialActivi
 	public boolean isUserliked(String userId, int targetId, int like) throws SocialActivityException {
```

Comment:
Thanks @Rajith90  , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/9ed86a227ed6dae6b761026c463faeb9935999d5)